Code/GraphMol/Chirality.cpp

```diff
@@ -1116,6 +1122,36 @@ void assignStereochemistry(ROMol &mol, bool cleanIt, bool force,
           (*bondIt)->getEndAtom()->getChiralTag() == Atom::CHI_UNSPECIFIED) {
         (*bondIt)->setBondDir(Bond::NONE);
       }
+
+      // make sure CIS/TRANS assignments are actually stereo bonds
```

Comment:
Having this code here doesn't work with dependent stereochemistry.
This snippet results in the stereo information on the first double bond:
```
    RWMol *m = SmilesToMol("FC=C(C/C=C/C)C/C=C\\C", false, false);
    TEST_ASSERT(m);
    MolOps::sanitizeMol(*m);

    TEST_ASSERT(m->getBondBetweenAtoms(1, 2));
    m->getBondBetweenAtoms(1, 2)->setStereoAtoms(0, 3);
    m->getBondBetweenAtoms(1, 2)->setStereo(Bond::STEREOCIS);

    {
      RWMol nm(*m);
      bool force = true, cleanIt = true;
      MolOps::assignStereochemistry(nm, cleanIt, force);

      nm.debugMol(std::cerr);
    }
```
I *think* I know how to fix this. I'll take a look.

Reply:
Well, that took considerably longer (and more code changes) than I had thought/hoped.
I'll submit a PR against your branch.